kyuubi-common/src/main/scala/org/apache/kyuubi/engine/result/TRowSetGenerator.scala

```diff
@@ -65,13 +61,11 @@ trait TRowSetGenerator[SchemaT, RowT, ColumnT]
     val tRowSet = new TRowSet(0, new JArrayList[TRow](rowSize))
     var i = 0
     val columnSize = getColumnSizeFromSchemaType(schema)
-    val tColumns = new JArrayList[TColumn](columnSize)
```

Comment:
it's by design, see https://github.com/apache/kyuubi/pull/5831

Reply:
ditto in row based implementation

Reply:
sorry for mistake , thanks.